UIDeviceIdentifier/UIDeviceHardware.m

```diff
@@ -22,113 +22,104 @@ + (NSString *) platform
     return platform;
 }
 
-+ (NSString *) platformString
-{
-    NSString *platform = [self platform];
-
-    NSDictionary *platformStrings = @{
++ (NSDictionary *) platformDefinitions {
+    NSDictionary *definitions = @{
 #if !defined(TARGET_OS_IOS) || TARGET_OS_IOS
-            @"iPhone1,1": @"iPhone 1G",
-            @"iPhone1,2": @"iPhone 3G",
-            @"iPhone2,1": @"iPhone 3GS",
-            @"iPhone3,1": @"iPhone 4 (GSM)",
-            @"iPhone3,2": @"iPhone 4 (GSM Rev A)",
-            @"iPhone3,3": @"iPhone 4 (CDMA)",
-            @"iPhone4,1": @"iPhone 4S",
-            @"iPhone5,1": @"iPhone 5 (GSM)",
-            @"iPhone5,2": @"iPhone 5 (GSM+CDMA)",
-            @"iPhone5,3": @"iPhone 5C (GSM)",
-            @"iPhone5,4": @"iPhone 5C (GSM+CDMA)",
-            @"iPhone6,1": @"iPhone 5S (GSM)",
-            @"iPhone6,2": @"iPhone 5S (GSM+CDMA)",
-            @"iPhone7,1": @"iPhone 6 Plus",
-            @"iPhone7,2": @"iPhone 6",
-            @"iPhone8,1": @"iPhone 6s",
-            @"iPhone8,2": @"iPhone 6s Plus",
-            @"iPhone8,4": @"iPhone SE",
-            @"iPhone9,4": @"iPhone 7 Plus",
-            @"iPhone9,2": @"iPhone 7 Plus",
-            @"iPhone9,3": @"iPhone 7",
-            @"iPhone9,1": @"iPhone 7",
-            @"iPhone10,1" : @"iPhone 8",
-            @"iPhone10,4" : @"iPhone 8",
-            @"iPhone10,2" : @"iPhone 8 Plus",
-            @"iPhone10,5" : @"iPhone 8 Plus",
-            @"iPhone10,3" : @"iPhone X",
-            @"iPhone10,6" : @"iPhone X",
-            @"iPhone11,2" : @"iPhone XS",
-            @"iPhone11,4" : @"iPhone XS Max",
-            @"iPhone11,6" : @"iPhone XS Max",
-            @"iPhone11,8" : @"iPhone XR",
-            @"iPod1,1": @"iPod Touch 1G",
-            @"iPod2,1": @"iPod Touch 2G",
-            @"iPod3,1": @"iPod Touch 3G",
-            @"iPod4,1": @"iPod Touch 4G",
-            @"iPod5,1": @"iPod Touch 5G",
-            @"iPod7,1": @"iPod Touch 6G",
-            @"iPod9,1": @"iPod Touch 7G",
```

Comment:
Why removing iPod Touch 7G?

Reply:
That wasn't intentional

Reply:
I mean that your whole list is missing plenty devices.
You should copy-paste it again from the current master branch.
Then squash your changes in a single commit.